core/tests/Test.Microsoft.Identity.Core.UIAutomation/Infrastructure/ITestController.cs

```diff
@@ -46,6 +46,11 @@ public interface ITestController
         /// <param name="isWebElement">Set to true if the element to be tapped is within a web view</param>
         void EnterText(string elementID, int waitTime, string text, bool isWebElement);
 
+        /// <summary>
+        /// Dismisses the keyboard
+        /// </summary>
+        void DismissKeyboard();
```

Comment:
>void DismissKeyboard(); [](start = 8, length = 23)

I did not see why this is needed in the pr description, but I can see that you have added the usage in several places in the test cases. Why is this needed?

Reply:
the android keyboard can sometimes hide buttons from the UI automation framework preventing it from clicking on buttons.

Reply:
Kindly ensure this is added to the remarks section of the xml comments

---
In reply to: [216706040](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1235#discussion_r216706040) [](ancestors = 216706040)